src/main/java/org/akaza/openclinica/controller/dto/SdvDTO.java

```diff
@@ -7,7 +7,7 @@
     private String participantId;
     private String siteName;
     private String eventName;
-    private Date eventStartDate;
+    private String eventStartDate;
```

Comment:
How come changing this to a string fixes the problem? Ideally, the dto should keep the date type and the javascript code should handle the conversion properly.

Reply:
hi Shiva, the Date arrives on the browser as int. Momentjs when given int will do timezone conversion, but won't if given String, so that fixed it. But OK, I'll revert the .java changes and handle all on client side.